libs/domain/auth/logout/src/logout.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { css } from 'lit';
+
+export const styles = css`
+  :host oryx-icon {
+    --oryx-icon-size: 24px;
+  }
+
+  button {
+    justify-content: flex-start;
+    padding: 9px 15px;
```

Comment:
button feels a bit huge, but I have not checked the figma

Reply:
Please check the designs. But from my point of view it's not so huge 
<img width="478" alt="image" src="https://user-images.githubusercontent.com/18655319/220571574-4fd8e4a7-767d-4c23-bc33-489a9177bbef.png">